Wikipedia/Code/WMFAccountLogin.swift

```diff
@@ -43,7 +43,7 @@ public class WMFAccountLoginResult: NSObject {
 }
 
 public class WMFAccountLogin: Fetcher {
-    public func login(username: String, password: String, retypePassword: String?, oathToken: String?, captchaID: String?, captchaWord: String?, siteURL: URL, success: @escaping WMFAccountLoginResultBlock, failure: @escaping WMFErrorHandler){
+    public func login(username: String, password: String, retypePassword: String?, oathToken: String?, captchaID: String?, captchaWord: String?, siteURL: URL, reattemptOn401Response: Bool = true, success: @escaping WMFAccountLoginResultBlock, failure: @escaping WMFErrorHandler){
```

Comment:
It seems that any login call shouldn't attempt re-login - if it failed with 401 once there's no reason to expect it'd succeed on a second try. Is there something I'm missing?